### DISCUSSION TITLE
Save Custom Prefix Prompt

### DISCUSSION THREAD

**Discussion by revbin** [danny-avila/LibreChat#394] [Feature Requests & Suggestions]
Hi

have the ability to save custom prompt prefix with Custom Name in DB and select them in UI if needed.

regards

**Comment by danny-avila** [comment]
This is possible already. Click "Save as preset"
![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/a895901e-a679-4026-932e-851970506708)

![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/91574cd8-9a20-451c-88b2-6598d4817109)
![image](https://github.com/danny-avila/chatgpt-clone/assets/110412045/f5384242-3927-4636-9c41-a47c8c1856b4)
